src/basic/socket-util.c

```diff
@@ -1238,21 +1239,22 @@ int flush_accept(int fd) {
 
                         return -errno;
 
-                } else if (r == 0)
```

Comment:
Not a matter, but I prefer to drop the above blank line.

Reply:
fixed